docs/contributing/999-rfcs/2024-01-15-console-sign-in.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Wing Console Sign In"
+description: Requirements for Wing Console Sign In process
+---
+
+# Wing Console Sign In
+
+- **Author(s)**: @ainvoner
+- **Submission Date**: 2024-01-15
+- **Stage**: Proposal
+- **Stage Date**: 2024-01-15
+
+This document describes the Wing Console Sign In process and rationale.
+
+## Background
+
+As we seek to find **Sales and Marketing Opportunities**, it has become increasingly important to establish a more direct and meaningful connection with our users.
+One of the ways to achieve this is by implementing a sign-in flow within the Wing Console application.
+
+## Requirements
+
+New users of Wing Console will be required to sign in with their GitHub account before having access to the Wing Console application.
+
+Another requirement is to have only one user entity in Wing. This means that a user in Wing Console is also a user in Wing Cloud and vice versa.
+
+## User Experience
+
+### New Users
+
+New users that will `wing it` will see a sign in dialog blocking the Wing Console UI. The rationale behind this decision is that we want to show the user a teaser of Wing Console while asking them to sign in.
+
+Once users will be signed in with their GitHub account they will be able to use Wing Console.
+
+### Singed In Users
+
+Signed in users to Wing Console will not need to sign in again. They will be able to use Wing Console as usual.
```

Comment:
Worth mentioning that the user is per machine so any time the console is opened, if there was already a logged in user, it will be used.

Reply:
Yes, the relationship of the analytics ID and the Wing Cloud user will only happen once. We won't require this process again anymore.